views/default/elements/components/river.css

```diff
@@ -13,6 +13,10 @@
 	border-bottom: 1px solid #e6e6ea;
 }
 
+.elgg-module .elgg-list-river > li:last-child {
+	border-bottom: 0;
```

Comment:
@hypeJunction is this what you mean?

Reply:
`.elgg-module .elgg-list > li:last-child {
   border-bottom: 0;
}
.elgg-module .elgg-list + .elgg-pagination {
  border-top: 1px solid;
}
`